todo_app/data/session_items.py

```diff
@@ -1,82 +1,173 @@
+
+
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN')
+   }
+
+   response = requests.request(
+      "GET",
+      url_for_board,
+      params=query
+   )
+   
+   getBoardresponse = response.text
+   getBoardresponse = json.loads(getBoardresponse)
+   return getBoardresponse[0]["id"]
+
+
+#########Get list id of To Do#########
+
+def gettodolistid(board_id):
+   
+   url_for_lists = 'https://api.trello.com/1/boards/'+board_id+'/lists'
+   
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN')
+   }
+
+
+   response = requests.request(
+      "GET",
+      url_for_lists,
+      params=query
+   )
+   
+   getListsresponse = response.text
+   getListsresponse = json.loads(getListsresponse)
+   
+   getListsresponse = [x['id'] for x in getListsresponse if x['name'] == 'To Do']
+   getListsresponse = getListsresponse[0]
+   return getListsresponse
+
+   
+##############Get list id of done############
+
+def getdonelistid(board_id):
+   
+   url_for_lists = 'https://api.trello.com/1/boards/'+board_id+'/lists'
+   
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN')
+   }
+
+   response = requests.request(
+      "GET",
+      url_for_lists,
+      params=query
+   )
+   
+   getListsresponse = response.text
+   getListsresponse = json.loads(getListsresponse)
+   
+   getListsresponse = [x['id'] for x in getListsresponse if x['name'] == 'Done']
+   getListsresponse = getListsresponse[0]
+   return getListsresponse
+
+
+################Get Items on To do and Done list ####################### 
+def getallitems ():
+   board_id = getBoardid()
+   todolistid = gettodolistid(board_id)
+   donelistid = getdonelistid(board_id)
+
+   return getToDoItems(todolistid,"Not Started") + getToDoItems(donelistid,"Done")
+
+def getToDoItems(list_id, list_name): 
+   
+   url_for_todoitems = 'https://api.trello.com/1/lists/'+list_id+'/cards'
+   
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN')
+   }
+
+   response = requests.request(
+      "GET",
+      url_for_todoitems,
+      params=query
+   )
+   
+   gettodoitems = response.text
+   gettodoitems = json.loads(gettodoitems)
+
+   result = []
+
+   for item in gettodoitems:
+      item1 = MyItem(item["id"],item["name"],list_name)
+      disp = item1.displayitem()
+      result.append(item1)
+   return result
+   
+######Create a card on To do items list
+
+def putToDoItems(newcardname):
+   board_id = getBoardid()
+   todolistid = gettodolistid(board_id)
+
+
+   url_for_todoitems = 'https://api.trello.com/1/cards'
+   
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN'),
+      'idList': todolistid,
+      'name' : newcardname
+
+   }
+
+   response = requests.request(
+      "POST",
+      url_for_todoitems,
+      params=query
+   )
+   
+   puttodoitems = response.text
+   puttodoitems = json.loads(puttodoitems)
```

Comment:
This code is never used, and though it might error if something doesn't work we shouldn't rely on that. If we want to check for failed requests, we could take advantage of Flask's `response.raise_for_status()`, otherwise we can remove these lines